website/blog/2022-11-30-databend-weekly-70.md

```diff
@@ -0,0 +1,185 @@
+
+- try fix lost tracker ([#8932](https://github.com/datafuselabs/databend/pull/8932))
+
+**Meta**
+
+- fix share db bug, create DatabaseIdToName if need ([#9006](https://github.com/datafuselabs/databend/pull/9006))
+
+**Mysql handler**
+
+- fix mysql conns leak ([#8894](https://github.com/datafuselabs/databend/pull/8894))
+
+**Processors**
+
+- try fix update list memory leak ([#9023](https://github.com/datafuselabs/databend/pull/9023))
+
+**Storage**
+
+- read and write block in parallel when compact ([#8921](https://github.com/datafuselabs/databend/pull/8921))
+
+## What's On in Databend
+
+Stay connected with the latest news about Databend.
+
+#### Preview of Infer Schema
+
+You usually need to create a table before loading data from a file stored on a stage or somewhere. Unfortunately, sometimes you might not know the file schema to create the table or are unable to input the schema due to its complexity. 
+
+Introducing the capability to infer schema from an existing file will make the work much easier. You will even be able to query data directly from a stage using a SELECT statement like `select * from @my_stage`.
+
+```sql
+INFER 's3://mybucket/data.csv' FILE_FORMAT = ( TYPE = CSV );
++-------------+---------+----------+
+| COLUMN_NAME | TYPE    | NULLABLE |
+|-------------+---------+----------|
+| CONTINENT   | TEXT    | True     |
+| COUNTRY     | VARIANT | True     |
++-------------+---------+----------+
+```
+
+We've added support for inferring the basic schema from parquet files in [#9043](https://github.com/datafuselabs/databend/pull/9043), and we're now working on [#7211](https://github.com/datafuselabs/databend/issues/7211) to implement `select from @stage`.
+
+**Learn More**
+
+- [PR | add basic schema infer for parquet](https://github.com/datafuselabs/databend/pull/9043)
+- [Issue | query data from S3 location or stage](https://github.com/datafuselabs/databend/pull/7211)
+- [PR | rfc: Infer Schema](https://github.com/datafuselabs/databend/pull/8645)
+
+## Issues
+
+Meet issues you may be interested in and try to solve it.
+
+#### Add tls support for mysql handler
+
+The just released [opensrv-mysql v0.3.0](https://github.com/datafuselabs/opensrv/discussions/35) includes support for tls and it is time to introduce it for Databend.
+
+```rust
+let (is_ssl, init_params) = opensrv_mysql::AsyncMysqlIntermediary::init_before_ssl(
+    &mut shim,
+    &mut r,
+    &mut w,
+    &Some(tls_config.clone()),
+)
+.await
+.unwrap();
+
+opensrv_mysql::secure_run_with_options(shim, w, ops, tls_config, init_params).await
+```
+
+[Issue 8983: Feature: tls support for mysql handler](https://github.com/datafuselabs/databend/issues/8983)
+
+If you find it interesting, try to solve it or participate in discussions and PR reviews. Or you can click on <https://link.databend.rs/i-m-feeling-lucky> to pick up a good first issue, good luck!
+
+## Changlogs
+
+You can check the changelogs of Databend nightly to learn about our latest developments.
```

Comment:
You can check the changelog of Databend Nightly for details about our latest developments.